flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/LocalExecutor.java

```diff
@@ -75,15 +69,13 @@ public class LocalExecutor implements Executor {
     private static final Logger LOG = LoggerFactory.getLogger(LocalExecutor.class);
 
     // result maintenance
-    private final ResultStore resultStore;
```

Comment:
Should we also need to store the results of multi jobs in memory like before?

Reply:
No. After this FLIP finishes, SqlGateway is responsible to store all job info. BTW, the SQL Client always works in a single thread. The current design introduces too much complexity.